benchmark.ttl

```diff
@@ -136,6 +140,20 @@
   rdfs:range xsd:integer;
   hobbit:defaultValue "100"^^xsd:integer .
 
+:blankNodesRatio a hobbit:Parameter, hobbit:ConfigurableParameter;
+  rdfs:label "Ratio of blank nodes"@en;
+  rdfs:comment "The ratio of blank nodes in the generated RDF graph."@en;
+  rdfs:domain hobbit:Experiment, hobbit:Challenge;
+  rdfs:range xsd:double;
+  hobbit:defaultValue "0.5"^^xsd:double .
+
+:literalsRatio a hobbit:Parameter, hobbit:ConfigurableParameter;
+  rdfs:label "Ratio of literals"@en;
+  rdfs:comment "The ratio of literals in the generated RDF graph."@en;
```

Comment:
The descriptions (`rdfs:comment`) could be a little bit more informative. It has been decided that the number of blank nodes and literals will be determined based on the number of nodes but that they will be added in addition to the normal RDF nodes. The description should express that. So maybe something like "Factor that is multiplied to the number of nodes to determine the number of literals that are added to the graph." That might be a little better than using the term "ratio", which sounds like it will take a part of the RDF nodes away to replace them with literals. :thinking: